cookbooks/vm/spec/integration/recipes/git_spec.rb

```diff
@@ -34,7 +38,7 @@
         st: 'status',
         unstage: 'reset HEAD --',
         slog: 'log --pretty=oneline --abbrev-commit',
-        graph: 'log --all --oneline --graph --decorate'
+        graph: 'log --all --oneline --graph --decorate',
```

Comment:
remove trailing comma?

Reply:
:-D bad habit from golang. There it is mandatory: https://dave.cheney.net/2014/10/04/that-trailing-comma

I'll remove it again.